services/ui-src/src/components/export/ExportedReportWrapper.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  Box,
+  Stack,
+  Table,
+  Thead,
+  Th,
+  Tr,
+  Td,
+  Tbody,
+  Text,
+} from "@chakra-ui/react";
+import {
+  ElementType,
+  FormPageTemplate,
+  MeasurePageTemplate,
+  ParentPageTemplate,
+} from "types";
+
+export const ExportedReportWrapper = ({ section }: Props) => {
+  const elements = section.elements
+    ?.filter(
+      (element) =>
+        element.type !== ElementType.ButtonLink &&
+        element.type !== ElementType.Accordion &&
+        element.type !== ElementType.Header &&
+        element.type !== ElementType.MeasureTable
```

Comment:
Alternate strategy here for musing, we could create an explicit map of a elements to pdf components. That might be a more long-term change, good solution here to get a page quickly online.

Reply:
Maybe some combination of the two where anything that passes through a known map we have built out gets the final component, and what remains gets a placeholder format here that we visually know looks like a placeholder.

Reply:
talked to @BearHanded, we're leaving it as is until we get a pdf design